src/uu/mv/src/mv.rs

```diff
@@ -210,15 +198,28 @@ fn determine_overwrite_mode(matches: &ArgMatches) -> OverwriteMode {
     }
 }
 
-fn exec(files: &[PathBuf], b: Behavior) -> i32 {
+fn exec(files: &[String], b: Behavior) -> i32 {
```

Comment:
why do you change it from PathBuf to String here?

Reply:
I can change it back, if you think that's best.  I thought we'd have to use canonical paths to solve this problem, and if converting filename strings to canonical paths threw an error, it should happen in the exec function where we can return with an error message and error code.  Turns out (I now think) the problem is much simpler.  We stat dirs correctly, we were just throwing a confusing/wrong error.

Reply:
I think what you did makes sense. However, I would propose to retrieve `OsString` from `clap` instead of `String`. I think this might be nice opportunity to change this (even though it already used `String` before this PR)

Reply:
Agreed.  Changed Strings to OsStrings as well.